src/views/RecIM/components/Forms/CreateTeamForm/index.js

```diff
@@ -122,9 +122,9 @@ const CreateTeamForm = ({
 
     let teamCreationRequest = { ...currentInfo, ...newInfo };
 
-    console.log(teamCreationRequest)
+    console.log(teamCreationRequest);
```

Comment:
Remove console logging here

Reply:
I'll make a note for the team. I only did a git pull from the current WIP branch